From fecf974b63f72eeb12d3b43522e948ca2bc704d4 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Thu, 11 Oct 2018 09:45:52 +0800
Subject: [PATCH] freeradius: correct version number of libssl defect

Upstream-Status: Backport [https://github.com/FreeRADIUS/freeradius-server
                 /commit/ad039347beca4ded297813a1da6eabb61fcf2ddd]

upstream have refactored this part code into
src/lib/tls/base.c, and problem also have fixed
by commit ad039347beca

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 src/main/tls.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/main/tls.c b/src/main/tls.c
index acbfe79..d9c91f1 100644
--- a/src/main/tls.c
+++ b/src/main/tls.c
@@ -72,15 +72,15 @@ typedef struct libssl_defect {
 static libssl_defect_t libssl_defects[] =
 {
 	{
-		.low		= 0x01010101f,		/* 1.1.0a */
-		.high		= 0x01010101f,		/* 1.1.0a */
+		.low		= 0x01010001f,		/* 1.1.0a */
+		.high		= 0x01010001f,		/* 1.1.0a */
 		.id		= "CVE-2016-6309",
 		.name		= "OCSP status request extension",
 		.comment	= "For more information see https://www.openssl.org/news/secadv/20160926.txt"
 	},
 	{
-		.low		= 0x01010100f,		/* 1.1.0  */
-		.high		= 0x01010100f,		/* 1.1.0  */
+		.low		= 0x010100000f,		/* 1.1.0  */
+		.high		= 0x01010000f,		/* 1.1.0  */
 		.id		= "CVE-2016-6304",
 		.name		= "OCSP status request extension",
 		.comment	= "For more information see https://www.openssl.org/news/secadv/20160922.txt"
-- 
2.7.4

